testing/src/cypress/page-objects/Institution-objects/BaseMethods.ts

```diff
@@ -1,5 +1,104 @@
+import data from "../../e2e/data/institution/manage-location.json";
+
+enum ErrorTypes {
+  fieldIsMandatory = 1,
+  fieldContainsMoreThanAllowedChars = 2,
+}
 export default class BaseMethods {
+  /**
+   *
+   * @param cyId data-cy id for the Element
+   * @returns Cypress.Chainable<JQuery<HTMLElement>>
+   */
   getElementByCyId(cyId: string) {
+    cy.get(`[data-cy='${cyId}']`).scrollIntoView();
     return cy.get(`[data-cy='${cyId}']`);
   }
+
+  /**
+   *
+   * @param fieldName Field Name that needs to be in the error message
+   * @param errorType What kind of error message to be constructed
+   * @returns The complete error message
+   */
+  errorMessageConstructor(fieldName: string, errorType: ErrorTypes) {
+    const fieldNameTrimmed = fieldName.trim();
+    switch (errorType) {
+      case 1:
```

Comment:
This can be replaced with the enum type as `case ErrorTypes.fieldIsMandatory`, right?